src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -1176,6 +1178,10 @@ protected String checkJavaVersion(final PrintStream logger, String javaCommand,
         return null;
     }
 
+    private double requiredJavaVersion() {
+        return Double.valueOf("1." + JDK_MAJOR_VERSION);
```

Comment:
Something like that. But I am not sure this plugin can wok on Java 6 due to the credentials dependency